src/main/scala/org/ergoplatform/nodeView/history/components/BlockSectionComponent.scala

```diff
@@ -10,12 +10,13 @@ import scala.util.Try
   * Trait that declares interfaces for validation and processing of various
   * block sections: BlockTransactions, ADProofs, etc.
   */
-trait BlockSectionProcessor extends ScorexEncoding {
+trait BlockSectionComponent extends ScorexEncoding {
```

Comment:
While it is natural to call an entity which is doing processing and validation "Processor", generic "Component" does not say much about what could be inside it (Section & Component sound like tautology also). Why this renaming then?